### ISSUE TITLE
Global Shortcuts Portal Support

### ISSUE THREAD

**Issue by KinteLiX** [tom-james-watson/Emote#88]
This would partly solve the issues with Wayland by allowing emote to register a shortcut for opening the emoji selection window.

**Comment by tom-james-watson** [comment]
I don't believe there's any way of doing this programmatically in Wayland (stupid, I know...). At least the last time I looked, which admittedly was a year or two ago, the only way of doing it is this - https://github.com/tom-james-watson/Emote/wiki/Hotkey-In-Wayland

**Comment by tom-james-watson** [comment]
Seems like this has been added to xdg-desktop-portal - https://flatpak.github.io/xdg-desktop-portal/#gdbus-org.freedesktop.portal.GlobalShortcuts. I'm not sure how that applies to this codebase, though, as it seems to be something related to flatpak. Honestly I think this app is going to just need to get rewritten from scratch to properly support flatpak and things like this.

**Comment by tom-james-watson** [comment]
Looks like flatpak is actually happening, so I think we should actually be able to do this over the next few months!

**Comment by gil0mendes** [comment]
Is it already possible to use the shortcut on Wayland `ctrl + alt + e`? I still not able to use it on NixOS with Gnome running on Wayland.

**Comment by tom-james-watson** [comment]
> Is it already possible to use the shortcut on Wayland `ctrl + alt + e`? I still not able to use it on NixOS with Gnome running on Wayland.

Have you followed https://github.com/tom-james-watson/Emote/wiki/Hotkey-In-Wayland?

**Comment by gil0mendes** [comment]
Oh, I missed that page 🤦 thanks, that worked perfectly.
